docs/competition/livestream/chat-moderation.md

```diff
@@ -0,0 +1,51 @@
+# Chat Moderation
+
+We want our chat to be casual, fun and welcoming to all.
```

Comment:
This would be a good place for something like "We actively encourage moderators to join the conversation in the chat and keep viewers engaged" as suggested in https://github.com/srobo/runbook/pull/113/files#r631314616

Reply:
687cfb2